### PR TITLE
Swap Nibbles in Java

### PR DESCRIPTION
**Pull request by ritvij14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
This is code to swap nibbles in Java. I have used Bitwise operators here.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue #980 
(Write your answer here.)

### CODE REVIEW

**Comment by ritvij14** [comment]
@iamrajiv @siddharth25pandey @Kajol-Kumari please review this PR

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]

File: Java/other/SwapNibbles.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+}
+/*
+* Sample input/output
+* Enter a number:
+* 100
+* Number after swapping nibbles:70
+*
+* Space and Time complexity = O(1)
+*/
```
Comment (Kajol-Kumari) [inline Java/other/SwapNibbles.java]: can we please have one blank line at the end here?

File: Java/other/SwapNibbles.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        int num = sc.nextInt();
+
+        int result = swapNibbles(num);
+
+        System.out.println("Number after swapping nibbles:" + result);
+        sc.close();
+    }
+
+    private static int swapNibbles(int x) {
+        // (x & 0x0F) << 4 gives us the last 4 bits of the number and
```
Comment (Kajol-Kumari) [inline Java/other/SwapNibbles.java]: Can we please use multi-line comment i.e `/* */` here from (line 17-20)?

**Review by Kajol-Kumari — APPROVED** [review]
@ritvij14 Minor changes required, Rest LGTM :+1:

File: Java/other/SwapNibbles.java (hunk 1 above)
Comment (ritvij14) [reply Java/other/SwapNibbles.java]: Done.

File: Java/other/SwapNibbles.java (hunk 2 above)
Comment (ritvij14) [reply Java/other/SwapNibbles.java]: @Kajol-Kumari done. Please have a look and merge if looks good.

**Comment by ritvij14** [comment]
@iamrajiv sir please merge this.

**Pull request merged**